packages/trace-viewer/src/ui/workbench.tsx

```diff
@@ -148,7 +148,7 @@ export const Workbench: React.FunctionComponent<{
 
   const revealAttachment = React.useCallback((attachment: AfterActionTraceEventAttachment) => {
     selectPropertiesTab('attachments');
-    setRevealedAttachment(attachment);
+    setRevealedAttachment({ ...attachment }); // copy to force re-render
```

Comment:
I'm not seeing why this clone is necessary, and in general causing rerenders in this manner is bad practice as it's difficult to track down why stuff is behaving as it is.

Reply:
It'll be passed to a `useEffect` down the line, so we need to do something to break `Object.is` equality.

Reply:
Revisiting this, you're right. Cloning an object is hard to debug because both objects will look the same. I've refactored it to use an incrementing counter instead.